docs/example-api-calls/google-workspace.md

```diff
@@ -1,235 +1,209 @@
 # API Call Examples for Google Workspace
 
-Example commands that you can use to validate proxy behavior against the Google Workspace APIs.
+Example commands (*) that you can use to validate proxy behavior against the Google Workspace APIs.
 Follow the steps and change the values to match your configuration when needed.
 
-To use, ensure you've set env variables on your machine:
-```shell
-# no slash at the end, and no function or lambda, just the base url of the deployment
-export PSOXY_BASE=https://YOUR_PSOXY_HOST
-export PSOXY_USER_TO_IMPERSONATE=you@acme.com
-```
+For GCP, you can use the `-i` flag to impersonate the desired user identity option when running the testing tool. Example:
 
-For GCP, use
 ```shell
-export OPTIONS="-g -i $PSOXY_USER_TO_IMPERSONATE"
+node tools/test-psoxy.js -u [your_psoxy_url]/psoxy-gcal/calendar/v3/calendars/primary -i you@acme.com
 ```
 
-For AWS, change the role to impersonate with one with sufficient permissions to call the proxy
+For AWS, change the role to impersonate with one with sufficient permissions to call the proxy (`-r` flag). Example:
+
 ```shell
-export AWS_ROLE_ARN="arn:aws:iam::PROJECT_ID:role/ROLE_NAME"
-export OPTIONS="-a -r $AWS_ROLE_ARN -i $PSOXY_USER_TO_IMPERSONATE"
+node tools/test-psoxy.js -u [your_psoxy_url]/psoxy-gcal/calendar/v3/calendars/primary -r arn:aws:iam::PROJECT_ID:role/ROLE_NAME
```

Comment:
the `[your_psoxy_url]` replacement for the customer throughout all the code seems a bit tedious. The env variable at least allows to copy-paste directly from here and run. I'd consider bringing it back.
Same for the role, not explicit in the calls anymore (before was part of the $OPTIONS).
What about the tool reading a `configuration.json` file with the proxy base and role?

Reply:
Yes, that will be part of the "playbook/test-suites"; actually, as `.env` file